### ISSUE TITLE
Function description inconsistence

### ISSUE THREAD

**Issue by helpse** [kohana/core#496]
File:
https://github.com/kohana/core/blob/3.3/master/classes/Kohana/Request.php

Function:
public function url()

**Comment by shadowhand** [comment]
You're right, this is not accurate:

```
* @param   array    $params    URI parameters
* @param   mixed    $protocol  protocol string or Request object
```

**Comment by kemo** [comment]
Fixed by e9d912da372900c6904c5131a8ea6fb10365978e
